R/fit_curves.R

```diff
@@ -487,7 +487,12 @@ logistic_metrics <- function(c, x_metrics) {
 #' @keywords fit_curves
 #' @export
 .setup_metric_output <- function() {
-  resp_metric_cols <- c(get_header("response_metrics"), "maxlog10Concentration", "N_conc")
+  resp_metric_all_cols <- c(get_header("response_metrics"), "maxlog10Concentration", "N_conc")
+  # remove cols ending with "_sd"
+  # they are not present in the primary assays 
+  # but only with the assays followed by averaging of biological replicates
+  resp_metric_cols <- grep("_sd$", resp_metric_all_cols, invert = TRUE, value = TRUE) 
```

Comment:
```suggestion
  resp_metric_cols <- resp_metric_all_cols[!endsWith(resp_metric_all_cols, "_sd")]
```

another solution to get rid of grep